npu.user.js

```diff
@@ -1044,7 +1044,7 @@
           '<style type="text/css"> ' +
             '#h_exams_gridExamList_bodytable tr.gridrow_blue td ' +
             '{ ' +
-              'background: #F8EFB1 !important; ' +
+              'background-color: #F8EFB1 !important; ' +
```

Comment:
The `[+]` icon next to the rows is implemented as a cell background image :facepalm:
This change prevents the icon from being removed from subscribed exam rows.

Reply:
@solymosi Could you please implement this fix to the "my own exams" page too? It seems that the ➕ buttons are missing from that page too.